### PR TITLE
#195: Fix migration of show-if/hide-if macro

### PR DESCRIPTION
**Pull request by Josue-T**
Implement migration of show-if and hide-if macro

cf #195

### CODE REVIEW

**Comment by Josue-T** [comment]
Hello,

@raphj can you review it :wink:

**Review by trrenty — APPROVED** [review]
looks good

File: application-confluence-migrator-pro-converters/src/main/java/com/xwiki/confluencepro/converters/internal/AbstractShowIfHideIfMacroConverter.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import org.xwiki.contrib.confluence.filter.internal.input.ConfluenceConverter;
+import org.xwiki.contrib.confluence.filter.internal.macros.AbstractMacroConverter;
+
+/**
+ * Converter used for hide-if and show-if macro.
+ *
+ * @version $Id$
+ * @since 1.23.2
+ */
+public abstract class AbstractShowIfHideIfMacroConverter extends AbstractMacroConverter
```
[the remark this answers is not in the document]
Comment (Josue-T) [reply application-confluence-migrator-pro-converters/src/main/java/com/xwiki/confluencepro/converters/internal/AbstractShowIfHideIfMacroConverter.java]: Thanks for the advice. Interesting solution.

**Comment by Josue-T** [comment]
We will probably can merge since we target the Confluence XML package to version `9.57.0`

**Pull request merged**